examples/BUILD

```diff
@@ -0,0 +1,38 @@
+load("//:bazel_versions.bzl", "SUPPORTED_BAZEL_VERSIONS")
+load(
+    "@cgrindel_rules_bazel_integration_test//bazel_integration_test:defs.bzl",
+    "bazel_integration_tests",
+    "default_test_runner",
+    "integration_test_utils",
+)
+
+default_test_runner(
+    name = "simple_test_runner",
+    bazel_cmds = [
+        "info",
+        "run //:buildifier.check",
+        "test //...",
+    ],
+)
+
+bazel_integration_tests(
+    name = "simple_test",
+    bazel_versions = SUPPORTED_BAZEL_VERSIONS,
+    test_runner = ":simple_test_runner",
+    workspace_files = integration_test_utils.glob_workspace_files("simple") + [
+        "//:local_repository_files",
+    ],
+    workspace_path = "simple",
+)
+
+test_suite(
+    name = "integration_tests",
+    # If you don't apply the test tags to the test suite, the test suite will
+    # be found when `bazel test //...` is executed.
```

Comment:
When testing locally it sounds nice to be able to use this single command to test everything, what problems does it cause?

Reply:
We can remove the tags if you would like the integration tests to always run. 

In my workflow, I tend to run `bazel test //...` very often. Since the integration tests take a bit longer to execute, I usually do that more sparingly by running `bazel test //:all_integration_tests`. This will execute all of the integration tests while I get some coffee. ☕️